src/components/resetpassword-form.js

```diff
@@ -0,0 +1,72 @@
+
+import React from "react";
+import { Input } from "./shared/input";
+
+export class ResetPassword extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentPassword: "",
+      password: "",
+      passwordConfirm: "",
+      touched: {
+        currentPassword: false,
+        password: false,
+        passwordConfirm: false,
+      }
+    };
+  }
+
+  onChangeInputText = e => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    });
+  }
+
+  handleBlur = (field) => (evt) => {
+    this.setState({
+      touched: { ...this.state.touched, [field]: true },
+    });
+  }
+
+  isValidPassword = () => {
+    const { password, passwordConfirm } = this.state;
+    if (password === "") {
+      return false;
+    }
+    if (password !== passwordConfirm) {
+      return false;
+    }
+    return true;
+  };
```

Comment:
Can you think of a way to simplify the above logic using boolean logic and truthy / falsy values in JS?

Reply:
:O i put a refactor.